apps/src/applab/CrosshairOverlay.js

```diff
@@ -95,6 +101,29 @@ CrosshairOverlay.prototype.render = function (intoElement, nextProps) {
   this.text_.setAttribute('x', textX);
   this.text_.setAttribute('y', textY);
   this.text_.textContent = this.getCoordinateText();
+
+  // If user is hovering over a control, show the element's id as a second tooltip
+  if (this.props_.mouseoverApplabControlId) {
```

Comment:
This doesn't quite duplicate the above, and in fact depends on it, so it might be fine for now.  We could try to extract the text bubble stuff into a sub-component now, or we could wait until we have three of them.  What do you think?